nginx.conf.sample

```diff
@@ -242,6 +242,6 @@ gzip_types
 gzip_vary on;
 
 # Banned locations (only reached if the earlier PHP entry point regexes don't match)
-location ~* (\.php$|\.phtml$|\.htaccess$|\.git) {
+location ~* (\.php$|\.phtml$|\.htaccess$|\.htpasswd$|\.git) {
```

Comment:
Could you please check if such implementation will work fine? If so - please apply it
```suggestion
location ~* (\.php$|\.phtml$|\.ht*|\.git) {
```

Reply:
@ihor-sviziev: your suggestion is a bit weird because the `*` modifier applies to the `t` character and means that the `t` should occur 0, 1 or multiple times, which yes, in theory it will match those two files, but also a lot more files. I believe the original suggestion is a lot better and more readable.

Or alternatively: we could choose to block all hidden files/directories, so all locations that start with a `.`, maybe that's also an idea? Something like: `(\.php$|\.phtml$|^\..+$)` maybe...

Reply:
@hostep,
You're right! Should be like this
```suggestion
location ~* (\.php$|\.phtml$|\.ht[a-z]+$|\.git) {
```